### PR TITLE
cassette: init at 0.2.0

### PR DESCRIPTION
**Pull request by AveryanAlex**
## Init cassette at 0.2.0

https://github.com/Rirusha/Cassette

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/ca/cassette/package.nix [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "Rirusha";
+    repo = "Cassette";
+    rev = "ver-${version}";
+    hash = "sha256-x9BRoLXrCO/7pI392MbO6m39rmpiOdCcp+pOLG6+k/s=";
+  };
+
+  nativeBuildInputs = [
```
Comment (AndersonTorres) [inline pkgs/by-name/ca/cassette/package.nix:34]: Put the elements in alphabetical order

File: pkgs/by-name/ca/cassette/package.nix [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    meson
+    ninja
+    vala
+    pkg-config
+    blueprint-compiler
+    desktop-file-utils
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [
```
Comment (AndersonTorres) [inline pkgs/by-name/ca/cassette/package.nix:44]: here too

File: pkgs/by-name/ca/cassette/package.nix [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    libgee
+    libxml2
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-bad
+    webkitgtk_6_0
+    glib-networking
+  ];
+
```
Comment (AndersonTorres) [inline pkgs/by-name/ca/cassette/package.nix:59]:
```suggestion

  strictDeps = true;

```

File: pkgs/by-name/ca/cassette/package.nix [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  meta = with lib; {
```
Comment (AndersonTorres) [inline pkgs/by-name/ca/cassette/package.nix:60]:
```suggestion
  meta = {
```

File: pkgs/by-name/ca/cassette/package.nix [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    description = "GTK4/Adwaita application that allows you to use Yandex Music service on Linux operating systems";
+    homepage = "https://github.com/Rirusha/Cassette";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ averyanalex ];
+    platforms = platforms.linux;
```
Comment (AndersonTorres) [inline pkgs/by-name/ca/cassette/package.nix:65]:
```suggestion
    license = lib.licenses.gpl3Plus;
    maintainers = with lib.maintainers; [ averyanalex ];
    platforms = lib.platforms.linux;
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

**Comment by averyanalex** [comment]
@AndersonTorres  I think review can be resolved

**Review by Aleksanaa — APPROVED** [review]

**Pull request merged**